package.json

```diff
@@ -3,11 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@comunica/actor-http-proxy": "^2.8.1",
     "@comunica/query-sparql": "^2.6.9",
-    "@solid/community-server": "^6.0.1",
     "@inrupt/solid-client": "^1.29.0",
     "@inrupt/solid-client-authn-browser": "^1.17.1",
     "@inrupt/vocab-common-rdf": "^1.0.5",
+    "@solid/community-server": "^6.0.1",
```

Comment:
Shouldn't this only be a dev dependency?

Reply:
Correct. This was an app dependency when trying to solve #51 , but I forgot to change it. It is now made a dev dependency in [commit 0088b1c](https://github.com/SolidLabResearch/generic-data-viewer/pull/72/commits/0088b1c8e0382449c0df06862b67c79031437903)